### PR TITLE
feat(cerebral-router): add routes config as option to router

### PR DESCRIPTION
**Pull request by christianalfoni**
I have added the routes config as an option to the router itself. I have kept the "routes by modules" to allow some more testing. Let us conclude after some discussion on it :)

### CODE REVIEW

File: packages/cerebral-router/src/index.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default function Router (options = {}) {
       throw new Error('Cerebral router - mapper option must be provided.')
     }
 
-    const routesConfig = flattenConfig(passedRoutesConfig)
+    const routesConfig = options.routes || flattenConfig(passedRoutesConfig)
```
Comment (Guria) [inline packages/cerebral-router/src/index.js]: Wouldn't it be frustrating that modules level config would be ignored when also routes passed to router itself?
I still feel sceptic about allowing modules level routes by default. It ok as extension but too opinionated for default

**Comment by Guria** [comment]
in #320 we suggested controller extension api.
my suggestion is to reimplement this special `router` field on modules as extension and doesn't ship it by default

File: packages/cerebral-router/src/index.js (hunk 1 above)
Comment (christianalfoni) [inline packages/cerebral-router/src/index.js]: Ah, okay, let me fix flattening

**Comment by christianalfoni** [comment]
Ah, yeah, that is a great idea :) Let me rebase this and remove the module routing stuff

**Comment by christianalfoni** [comment]
Okay, I have removed module routes config

**Pull request merged**